exchange/src/main/java/org/geneontology/gocam/exchange/BioPaxtoGO.java

```diff
@@ -1524,20 +1526,20 @@ else if (entity instanceof Interaction){
 	private IRI getPhysicalEntityIRI(Entity entity) {
 		String entity_id = getEntityReferenceId(entity);
 		if(entity_id!=null) {
+			if(entity_id.startsWith("CHEBI")) {
+				IRI entity_class_iri = IRI.create("http://purl.obolibrary.org/obo/"+entity_id);
+				if(golego.chebi_roles.contains(entity_class_iri.toString())) {
+					// No roles of any kind can be treated as physical entities - fall back on 'chemical entity'
+					return IRI.create("http://purl.obolibrary.org/obo/CHEBI_24431");
```

Comment:
@dustine32 it would be nice not to lose the role, but I guess we don't really want to have class expressions in GO-CAMS (it would be like `'chemical entity' and 'has role' some whatever`). We have terms for entity versions of the roles in GO (all the GOCHE IDs). You could maybe check if there is an appropriate GOCHE ID.

Reply:
@balhoff Huh, funny that this change shows as new in this PR. It should already have been in `master` from mega PR https://github.com/geneontology/pathways2GO/pull/158.

I did spot-check one of the example roles for the ticket https://github.com/geneontology/pathways2GO/issues/139: acceptor [CHEBI:15339](https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:15339). Should the GOCHE xrefs be visible in OLS? Or, how can a check whether `acceptor` has an entity version ChEBI?